types/slate/slate-tests.ts

```diff
@@ -282,3 +282,36 @@ editor
 KeyUtils.setGenerator(() => "Test");
 KeyUtils.create();
 KeyUtils.resetGenerator();
+
+Schema.create({
```

Comment:
This is problematic since there is no `Schema` class anymore...probably means there's a little more work around that

Reply:
Good catch; thanks. I've now changed the test to use the `SchemaProperties` type instead.